cli/cycode.py

```diff
@@ -141,5 +154,12 @@ def _should_fail_scan(context: click.Context):
     return scan_utils.is_scan_failed(context)
 
 
+def _sca_scan_to_context(context: click.Context, sca_scan_user_selected: List[str]):
+    for sca_scan_option in config['scans']['supported_sca_scans']:
+        context.obj[sca_scan_option] = None
```

Comment:
no need to init it. in dict when trying to get value of not exist key the value will be None

Reply:
But I try to access to it here:
`def get_default_scan_parameters(context: click.Context) -> dict:
    return {
        "monitor": context.obj["monitor"],
        "package_vulnerabilities": context.obj["package-vulnerabilities"],
        "license_compliance": context.obj["license-compliance"]
    }`